rustv1/premium-ex.md

```diff
@@ -0,0 +1,11 @@
+//Add paths for premium examples here which will be consumed by workflow to update KB with quality code
+//Don't forget to add new line at the end of this file 
+
+## basics:
+/example_code/controltower
```

Comment:
These paths don't look like Rust paths, are we sure they are correct?

Reply:
I updated the rust paths now